moduleframework/common.py

```diff
@@ -469,7 +409,7 @@ def get_template(self):
         get location of template from config.yaml file
         :return: str
         """
-        return self.info.get(TEMPLATE)
+        return self.info.get(conf["openshift"]["template"])
```

Comment:
Default value is missing.

Reply:
@jscotka Do you count with None if `template` is not specified?

Reply:
It has to be specified in this config. There is stored default value and in case user remove this item from config, it causes traceback, it is expected.